### PR TITLE
KRACOEUS-8523

### PR DESCRIPTION
**Pull request by rmancher**
KRACOEUS-8523
Blanket approve merged protocol document - IRB and IACUC.
Issue in this JIRA is that the new document not approved.
Status was ENROUTE not FINAL
Fixed in both IRB and IACUC

### CODE REVIEW

**Comment by jhulslander** [comment]
+1

File: coeus-impl/src/main/java/org/kuali/kra/protocol/ProtocolDocumentBase.java [hunk 1]
```diff
@@ -206,7 +205,16 @@ else if (isDisapproved(statusChangeEvent)) {
...
     
+    protected DocumentRouteHeaderValue getDocumentRouteHeaderValue() {
+        return KcServiceLocator.getService(RouteHeaderService.class).getRouteHeader(
+                this.getDocumentHeader().getWorkflowDocument().getDocumentId());
+    }
   
+    protected void setProtocolDocumentToApproveByDefault() {
+        DocumentRouteHeaderValue document = getDocumentRouteHeaderValue();
+        document.getDocumentType().getDefaultApprovePolicy().setPolicyValue(true);
+    }
```
Comment (gathreya) [inline coeus-impl/src/main/java/org/kuali/kra/protocol/ProtocolDocumentBase.java]: @rmancher this has always been false for protocol right? Why is this change needed now?

Comment (leoherbie) [inline coeus-impl/src/main/java/org/kuali/kra/protocol/ProtocolDocumentBase.java]: Can't we do this in the workflow xml?

Comment (rmancher) [inline coeus-impl/src/main/java/org/kuali/kra/protocol/ProtocolDocumentBase.java]: It is only for the merged Protocol document (amendment/renewal) where we need to blanket approve the new protocol document.

Comment (rmancher) [inline coeus-impl/src/main/java/org/kuali/kra/protocol/ProtocolDocumentBase.java]: @leoherbie as mentioned earlier, it is only for this specific scenario where we merge the amendment/renewal details to the new document once amendment/renewal is approved and we blanket approve the new document.

Comment (gathreya) [inline coeus-impl/src/main/java/org/kuali/kra/protocol/ProtocolDocumentBase.java]: @rmancher OK but was this not working in 5.2.1 then?

Comment (jhulslander) [inline coeus-impl/src/main/java/org/kuali/kra/protocol/ProtocolDocumentBase.java]: this does work in 5.2 (dly)

Comment (rmancher) [inline coeus-impl/src/main/java/org/kuali/kra/protocol/ProtocolDocumentBase.java]: @leoherbie yes it is working in 5.2.1
I'm not sure if anything changed with the default blanket approval policy with new rice upgrade.

**Comment by gathreya** [comment]
OK, I do see the rice change that has necessitated the change of the approval policy. I am OK with these changes.

**Comment by gathreya** [comment]
will merge if I do not hear from anyone else.
